about/people.yml

```
@@ -439,3 +439,12 @@ data:
     brown_directory_uuid: 542a2511-9c28-4f0d-a20b-107e002929fb
     bio: Melinda is a summer research intern working with the Computational Biology Core at CCV. She is an undergraduate student at Brown concentrating in Computational Biology. She loves matcha lattes and long walks in Providence.
     image: melinda_main.jpg
+    name: Jose Harvey
+    type: Part Time
```

I believe you need to add in the hyphen so it knows it's a separate entry. Right now you've overridded Melinda's entry 

```suggestion
  - name: Jose Harvey
    type: Part Time
```

---

Check out a YAML tutorial about arrays